### PR TITLE
Update Remove-AzApplicationGatewaySslCertificate.md

### PR DESCRIPTION
**Pull request by asukumar22**
Updated $AppGW to $AppGw which is a typo, and also updated the purpose of the last command "Set-AzApplicationGateway" as it was unclear in the document, sometimes this command might be missed because of lack of details, which might not save changes to the application gateway if not performed/missed.

## Checklist

- [x] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** select appropriate branch. Cmdlets from Autorest.PowerShell should go to [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch. 
* **SHOULD** make the title of PR clear and informative, and in the present imperative tense. 
* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. Add changelog in description section if PR goes into [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** have approved design review for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) ([_Microsoft internal only_](../blob/main/CONTRIBUTING.md#onboarding)) with following situations
    * Create new module from the scratch
    * Create new resource types which are not easy to conform to [Azure PowerShell Design Guidelines](../blob/main/documentation/development-docs/design-guidelines)
    * Create new resource type which name doesn't use module name as prefix
    * Have design question before implementation
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** introduce [breaking changes](../blob/main/documentation/breaking-changes/breaking-changes-definition.md) in Az minor release except preview version.
* **SHOULD NOT** change version of module in pull request

### CODE REVIEW

File: src/Network/Network/help/Remove-AzApplicationGatewaySslCertificate.md [hunk 1]
```diff
@@ -30,9 +30,9 @@ Remove-AzApplicationGatewaySslCertificate -ApplicationGateway $AppGW -Name "Cert
 Set-AzApplicationGateway -ApplicationGateway $AppGw
 ```
 
-The first command gets the application gateway named ApplicationGateway01 and stores the result in the variable named $AppGW.
-The second command removes the SSL certificate named Cert02 from the application gateway stored in the $AppGW variable.
-The last command updates the application gateway.
+The first command gets the application gateway named ApplicationGateway01 and stores the result in the variable named $AppGw.
+The second command removes the SSL certificate named Cert02 from the application gateway stored in the $AppGw variable.
```
Comment (BethanyZhou) [inline src/Network/Network/help/Remove-AzApplicationGatewaySslCertificate.md:34]: Please check L28, there is $AppGW in example 1

**Review by isra-fel — CHANGES_REQUESTED** [review]
Hi @asukumar22 please also fix the casing in line 28&29

**Review by asukumar22 — COMMENTED** [review]
The changes look correct to me.

**Comment by asukumar22** [comment]
> Hi @asukumar22 please also fix the casing in line 28&29

This is done with my recent upcoming pull request. Please merge that too. Thanks, Bethany!

**Review by BethanyZhou — APPROVED** [review]

**Review by isra-fel — APPROVED** [review]

**Pull request merged**